config/routes.rb

```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
+  root 'users#index'
+
+  get    'index'  => 'users#index' 
+  get	   'signup' => 'users#new'
+  get    'login'  => 'sessions#new'
+  post   'login'  => 'sessions#create'
+  delete 'logout' => 'sessions#destroy'
```

Comment:
substituir as linhas acima por 
```ruby
resource :sessions, only: [:new, :create, :destroy]
```

depois vc pode conferir as rotas utilizando `rake routes` no console

Reply:
Me da uma luz, to a um dia com isso. rs Eu olhei no guides, não consegui esclarecer uma coisa. Eu substituo todo mundo por esse cara " resource :sessions, only: [:new, :create, :destroy] ", mas como fica os nomes das rotas ?   

http://guides.rubyonrails.org/routing.html#singular-resources